server/local/studentdashboard-DEV.yml

```diff
@@ -1,13 +1,8 @@
 ### This yaml format file specifies configuration values that may change from
```

Comment:
Why a total of 5 *DEV*.yml files? Seems like we'd only need a restrictive and permissive ones -- otherwise seems like a maintenance chore (with room for error) to keep them all up-to-date.